packages/ec-core/Cargo.toml

```diff
@@ -33,6 +33,8 @@ rand = { workspace = true, features = ["alloc", "small_rng"] }
 criterion = { workspace = true }
 miette = { workspace = true, features = ["fancy"] }
 thiserror = { workspace = true }
+proptest.workspace = true
+test-strategy = "0.4.0"
```

Comment:
don't we have test-strategy in the workspace yet? If we do not we should move it there and change all occurrences in dependencies to use that ( which I expect to be exactly one in that case )

Reply:
Duh – good catch. `test-strategy` is definitely already in the top level `Cargo.toml`, and I should use `workspace = true` here.